wiki/People/Tournament_Committee/en.md

```diff
@@ -26,7 +33,111 @@ Their responsibilities include:
+- Reach out to the infringing party, e.g. the tournament organiser, to get further information on the events detailed in the report
+- If after this investigation is over and the Tournament Committee is still considering handing out relevant punishment, hold an anonymous vote to determine whether to do so and the severity of such punishment
+
+### News Posts
+
+When an email requesting registration and conclusion news posts for a tournament is received, a thread is automatically created with an embed message of the email’s text. Any Tournament Committee member who sees this will ping the rest of the group for visibility. The committee will then check if the tournament is eligible based on the requirements on the Official Support wiki page before moving forward. If the tournament is eligible and the consensus in discussion around the tournament is positive after 48 hours, the Tournament Committee will forward the request to the news post team and approve the request. If the tournament does not pass one of these steps, the request will be denied.
+
+### Top Three Badge Support
+
+If a tournament host believes their tournament has achieved the highest level of production quality and organisation possible, they can request to receive badge prizes for all three finalists in their tournament. If a host requests to be awarded with this distinction, then in addition to the typical steps for tournament review as listed above, the assigned members will also notify the rest of the group of the request for top three badge support. Based on discussion around the tournament and its quality, either the request will be denied, or an anonymous vote will be called for the tournament to receive top three badge support.
+
+### Public Communication
+
+Public communication by the Tournament Committee is handled through several channels. The purposes for these sometimes overlap, but all are official sources of information kept up-to-date by the Tournament Committee.
+
+The Tournament Committee maintains an [account on X](https://twitter.com/osu_tcomm) (formerly Twitter) where the committee broadcasts major changes and updates of various types. Not all changes are communicated here, but many of the most important changes are communicated and clarified through this channel.
+
+The [Community Tournament Status Tracker](https://docs.google.com/spreadsheets/d/1bV5MyrJZI0F52Bx9EcYxdfRu8qGnhWRBKdXHK9uPbO0/edit?usp=sharing) is a publicly viewable spreadsheet detailing the status of all tournaments that have requested official support, i.e. all tournaments that requested screening. This sheet shows all tournaments dating back to January 14th, 2023, and contains a description of the status of the tournament at any given time, including whether a badge was applied or denied. If a badge was denied, the reason for denial is stated on the sheet.
```

Comment:
```suggestion
The [Community Tournament Status Tracker](https://docs.google.com/spreadsheets/d/1bV5MyrJZI0F52Bx9EcYxdfRu8qGnhWRBKdXHK9uPbO0/edit?usp=sharing) is a publicly viewable spreadsheet detailing the status of all tournaments that have requested official support. This sheet shows all tournaments dating back to June 20th, 2023 and displays their status through the official support process, including whether a badge was applied or denied.
```

Date given by Albion, apparently not *all* tournaments from January-June 20th were on this sheet.